app/code/Magento/Ui/view/base/web/js/modal/prompt.js

```diff
@@ -61,8 +61,8 @@ define([
                 /**
                  * Click handler.
                  */
-                click: function () {
-                    this.closeModal(true);
+                click: function (event) {
+                    this.closeModal(event);
```

Comment:
Should the result be passed as well?
```suggestion
                    this.closeModal(event, true);
```

Reply:
I have added condition if result is not passed then result will take as false